nilearn/datasets/tests/test_utils.py

```diff
@@ -382,7 +382,7 @@ def test_uncompress_zip(tmp_path):
     # and check if ftemp exists
     ztemp = tmp_path / "test.zip"
     ftemp = "test"
-    with contextlib.closing(ZipFile(ztemp, "w")) as testzip:
+    with ZipFile(ztemp, "w") as testzip:
```

Comment:
`Zipfile` is already a context manager: https://docs.python.org/3/library/zipfile.html#zipfile-objects